tests/system/action/organization/test_update.py

```diff
@@ -20,6 +20,7 @@ def test_update_some_more_fields(self) -> None:
         self.create_model(
```

Comment:
Please write tests for filling/changing theme_id and theme_ids. Show that it is not possible to remove the default theme from theme_id and theme_ids

Reply:
Changed.